packages/lexical-editor/src/ui/DropDown.tsx

```diff
@@ -162,42 +166,70 @@ export function DropDown({
 
     useEffect(() => {
         const button = buttonRef.current;
-        if (button === null || !showDropDown) {
-            return;
+        const dropDown = dropDownRef.current;
+
+        if (showDropDown && button && dropDown) {
+            dropDown.style.top = "44px";
+            dropDown.style.left = `${button.offsetLeft}px`;
+            setPositionIsCalculated(true);
         }
+    }, [dropDownRef, buttonRef, showDropDown]);
 
-        const handle = (event: MouseEvent) => {
-            const target = event.target;
-            if (!button.contains(target as Node)) {
-                setShowDropDown(false);
-            }
-        };
-        document.addEventListener("click", handle);
+    // @ts-ignore - it's ok that all paths do not return value
```

Comment:
We still have TS ignore here?

TS is complaining here and it's a for a good reason. Let's fix the error by adding the return statement where needed.

I have to repeat - using @ts-ignore is just for cases where it's really tricky to fix, or fixing would take unreasonable amount of time. This is a trivial TS issue, let's address it properly.